Dockerfile-gin

```diff
@@ -0,0 +1,52 @@
+FROM golang:1.19.5-alpine AS build
```

Comment:
Please rename this to Dockerfile-taas as we are now moving over to this terminology

Reply:
Updated the docker file name to Dockerfile-taas